utils/progressbar/transferprogressbarmanager.go

```diff
@@ -43,13 +47,13 @@ func formatString(emoji, key string, windows bool) string {
 
 func initSProgressBarLabels(windows bool) transferLabels {
 	pbs := transferLabels{}
-	pbs.RetryFailureContentNote = "In Phase 3 and in subsequent executions, we'll retry transferring the failed files."
```

Comment:
Was this line removed intentionally?

Reply:
Yes, it is not smart to store const text in a class variable